### ISSUE TITLE
I can't install this lib via go mod, git doesn't have permission to pull the dependency

### ISSUE THREAD

**Issue by GargantuaX** [weaviate/weaviate-go-client#142]
I can't install this lib via go mod, git doesn't have permission to pull the dependency:

github.com/weaviate/weaviate@v1.18.1-0.20230307173515-39ef7cc0c9ff

```shell
go: downloading github.com/milvus-io/milvus-proto/go-api v0.0.0-20230301092744-7efc6eec15fd
go: downloading google.golang.org/grpc/examples v0.0.0-20220617181431-3e7b97febc7f
go: finding module for package github.com/weaviate/weaviate-go-client/v4/weaviate
go: found github.com/weaviate/weaviate-go-client/v4/weaviate in github.com/weaviate/weaviate-go-client/v4 v4.6.3
go: github.com/weaviate/weaviate-go-client/v4@v4.6.3 requires
	github.com/weaviate/weaviate@v1.18.1-0.20230307173515-39ef7cc0c9ff: reading https://goproxy.io/github.com/weaviate/weaviate/@v/v1.18.1-0.20230307173515-39ef7cc0c9ff.mod: 404 Not Found
	server response: not found: github.com/weaviate/weaviate@v1.18.1-0.20230307173515-39ef7cc0c9ff: invalid version: unknown revision 39ef7cc0c9ff
```

**Comment by GargantuaX** [comment]
It works, thank you.

I do this, share for other one.

```shell
// add to go.mod

replace github.com/weaviate/weaviate-go-client/v4 => github.com/weaviate/weaviate-go-client/v4 v4.6.4-0.20230324165741-697635d9b99a
```

**Comment by dirkkul** [comment]
Thanks for confirming! I released v4.6.4 which includes the fix, might be the best to update to that version
